### PR TITLE
feat: Added announcements

### PR DESCRIPTION
**Pull request by gbdlin**
Added announcements (also known as motd) that will be used by decky to show important messages to users.

### CODE REVIEW

File: plugin_store/api/utils.py [hunk 1]
```diff
@@ -35,3 +36,7 @@ def __init__(self, model: Any = None, *, use_cache: bool = True):
 # noinspection PyPep8Naming
 def FormBody(model: Any = None, *, use_cache: bool = True) -> Any:
     return FormBodyCls(model=model, use_cache=use_cache)
+
+
+class UUID7(UUID1):
```
Comment (suchmememanyskill) [inline plugin_store/api/utils.py:41]: This seems a little odd imo. Whats wrong with uuid.uuid4()

File: plugin_store/database/database.py [hunk 2]
```diff
@@ -67,6 +69,55 @@ def init(self):
         alembic_cfg = Config("/alembic.ini")
         command.upgrade(alembic_cfg, "head")
 
+    async def list_announcements(self, active: bool = True):
```
Comment (suchmememanyskill) [inline plugin_store/database/database.py:72]: Unusued var

**Review by suchmememanyskill — APPROVED** [review]

File: plugin_store/api/utils.py (hunk 1 above)
Comment (gbdlin) [reply plugin_store/api/utils.py:41]: v4 are completely random, and DB primary indexes don't like random values. v7 is designed to be monothonic, which is much better suited for indexing. The implementation of v7 generator comes directly from Python 3.14 source code, I just backported it here as 3.14 is not yet released.

This is probably an overkill for announcements, as there shouldn't be many of them in DB at the same time (at least for now as we don't have a way of hiding them without deleting), but it's a good spot to test them out initially before introducing them for plugins.

File: plugin_store/database/database.py (hunk 2 above)
Comment (gbdlin) [reply plugin_store/database/database.py:72]: Yes, it's unused. i wanted to define the API of that function for future use to differentiate between 2 different endpoints that will use it. In future it'll add filters that will only list announcements that are currently live, that is enabled and/or in a specific timeframe.

File: plugin_store/database/models/announcements.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from .Base import Base
+
+UTC = ZoneInfo("UTC")
+
+
+def utcnow() -> datetime:
+    return datetime.now(UTC)
+
+
+class Announcement(Base):
```
Comment (AAGaming00) [inline plugin_store/database/models/announcements.py:18]: Maybe add some way to mark an announcement as inactive so we can keep old ones around?

Comment (gbdlin) [reply plugin_store/database/models/announcements.py:18]: Wanted to reserve it for future, but maybe it's a good idea to do it now. Added.

**Pull request merged**